mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -24,21 +24,24 @@
 	</h2>
 
 	<div class="task-container">
-		<aui:input cssClass="input-item" name="task" />
+		<aui:input cssClass="input-item" name="task" placeholder="Enter your task.." />
 
-		<aui:button cssClass="add-task" name="add" />
+		<aui:button cssClass="add-task" name="add" value="add" />
 
-		<span class="counter">30</span>
+		<span class="counter">40</span>
 
 		<liferay-ui:message key="characters-remaining" />
 
 		<ul class="task-list">
 			<li>
 				<liferay-ui:message key="sample-task" />
-
-				<i class="icon-remove"></i>
+				<aui:button cssClass="delete-task" icon="icon-remove" size="large" />
 			</li>
 		</ul>
+		<div class="line-separator">
```

Comment:
There is no need for this ```<div>```. You can make a 1px line with border style of sibling element.